platform/core/src/services/ViewportGridService/getPresentationIds.ts

```diff
@@ -109,8 +115,30 @@ const getPresentationIds = (viewport, viewports): PresentationIds => {
     lutPresentationArr.push(uid);
   }
 
-  addUniqueIndex(positionPresentationArr, 'positionPresentationId', viewports);
-  addUniqueIndex(lutPresentationArr, 'lutPresentationId', viewports);
+  // only add unique index if the viewport is getting inserted and not updated
+  const isUpdatingViewport = viewports.find(v => {
```

Comment:
The name of this variable suggests that it is a boolean, but instead it is a `Viewport`. So either change its name or use `viewports.some`